src/components/BooksUI/Bookinput.js

```diff
@@ -41,17 +41,18 @@ const Bookinput = (() => {
 
   return (
 
-    <form onSubmit={submitBookToStore}>
-      <input name="title" type="text" placeholder="Enter book name" value={newTitle.title} onChange={titleChange} />
-      <select name="category" onChange={categoryChange} value={newCategory.category} id="myCategory" placeholder="Category">
+    <form onSubmit={submitBookToStore} className="inputForm">
+      <h2 className="Title">ADD NEW BOOK</h2>
+      <input name="title" type="text" placeholder="Enter book name" value={newTitle.title} onChange={titleChange} className="Lesson-Pane2 titling" />
+      <select name="category" onChange={categoryChange} value={newCategory.category} id="myCategory" placeholder="Category" className="Lesson-Pane2">
         <option value="">Category</option>
         <option value="Action">Action</option>
         <option value="Horror">Horror</option>
         <option value="Science Fiction">Science Fiction</option>
         <option value="Economy">Economy</option>
         <option value="Comedy">Comedy</option>
       </select>
-      <button type="submit">Add</button>
+      <button type="submit" className="submit">Add Book</button>
     </form>
   );
```

Comment:
- [ ] Kindly, compare your implementation with the design provided to meet all the requirements. Please use the same font size, color, and font family.

Original design

![header--2](https://user-images.githubusercontent.com/55635977/150629230-3a22833c-702e-4224-85b5-e52d9688e667.png)

Your design

![header--6](https://user-images.githubusercontent.com/55635977/150629207-509b3323-2b6c-4577-a212-ff828d5d847c.png)